### PR TITLE
Fixes #6741: rudder-dev commit fails to assign tickets for non-admin users

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/6741

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

File: scripts/rudder-dev/rudder-dev [hunk 1]
```diff
@@ -423,41 +424,62 @@ def ticket_to_in_progress(ticket_id, message=None):
...
+  else:
+    users = { 'nrm_users': {} }
+  # just update it otherwise we would have only project specific data in it
+  user_list = requests.get(REDMINE_API_URL + "/projects/" + str(project_id) + "/memberships.json", headers = {'X-Redmine-API-Key': REDMINE_TOKEN } )
+  for user in user_list.json()['memberships']:
+    id = user['user']['id']
+    user_data = requests.get(REDMINE_API_URL + "/users/" + str(id) + ".json", headers = {'X-Redmine-API-Key': REDMINE_TOKEN } )
+    user_detail = user_data.json()['user']
+    if user_detail['mail'].endswith("normation.com"):
+      trigraph = (user_detail['firstname'][0] + user_detail['lastname'][0] + user_detail['lastname'][1]).upper()
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: Maybe you can replace getting the first two character of lastname by using: 

user_detail['lastname'][:2]

see: http://stackoverflow.com/questions/11714859/how-to-display-the-first-few-characters-of-a-string-in-python

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**